introduction/examples/thompson/fibonacci.py

```diff
@@ -0,0 +1,48 @@
+def list_fibonacci(n):
+    if isinstance(n, int):
+        if n >= 0:
+            list1 = [0]
+            pre = 0
+            post = 1
+            for i in range(n):
+                tmp = pre + post
```

Comment:
Python jest na tyle fajny, że zamiany (swap'y) dwóch czy więcej wartości można robić w miejscu. Jest to pochodna możliwości które oferują krotki.
`pre, post = post, pre + post`